src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/scheduler/PrmSchedulerProcess2IntegrationTest.java

```diff
@@ -0,0 +1,290 @@
+            "/SchedulerTests/PrdOrganisationInfo/organisation3_scenario_01.json"
+        ), Integer.valueOf(TEST_PAGE_SIZE));
+
+        // verify that the ProfileRefreshQueue contains the expected OrganisationProfileId and set to inactive
+        assertProfileRefreshQueueEntityInDb(SOLICITOR_PROFILE, 1, false);
+
+        // verify that the OranisationRefreshQueue contains the expected OrganisationProfileId and set to active
+        assertOrganisationRefreshQueueEntitiesInDb("1", 1, true, NEW_ORGANISATION_LAST_UPDATED, true);
+        assertOrganisationRefreshQueueEntitiesInDb("2", 1, true, NEW_ORGANISATION_LAST_UPDATED, true);
+        assertOrganisationRefreshQueueEntitiesInDb("3", 1, true, NEW_ORGANISATION_LAST_UPDATED, true);
+    }
+
+    /**
+     * New Organisations (PageSize=1, Pages=3) - Insert organisations to an empty list.
+     */
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {
+        "classpath:sql/prm/profile_refresh_queue/init_profile_refresh_queue.sql",
+        "classpath:sql/prm/profile_refresh_queue/insert_Solicitor_Profile.sql",
+        "classpath:sql/prm/organisation_refresh_queue/init_organisation_refresh_queue.sql"
+    })
+    void testOrgChangeExistingProfilePageSize1() {
+
+        // verify that the Organisations are updated
+        runTest(List.of(
+            "/SchedulerTests/PrdOrganisationInfo/organisation1_scenario_01.json",
+            "/SchedulerTests/PrdOrganisationInfo/organisation2_scenario_01.json",
+            "/SchedulerTests/PrdOrganisationInfo/organisation3_scenario_01.json"
+        ), 1);
+
+        // verify that the ProfileRefreshQueue contains the expected OrganisationProfileId and set to inactive
+        assertProfileRefreshQueueEntityInDb(SOLICITOR_PROFILE, 1, false);
+
+        // verify that the OranisationRefreshQueue contains the expected OrganisationProfileId and set to active
+        assertOrganisationRefreshQueueEntitiesInDb("1", 1, true, NEW_ORGANISATION_LAST_UPDATED, true);
+        assertOrganisationRefreshQueueEntitiesInDb("2", 1, true, NEW_ORGANISATION_LAST_UPDATED, true);
+        assertOrganisationRefreshQueueEntitiesInDb("3", 1, true, NEW_ORGANISATION_LAST_UPDATED, true);
+    }
+
+    /**
+     * New Organisations - Insert organisations to an already populated list.
+     */
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {
+        "classpath:sql/prm/profile_refresh_queue/init_profile_refresh_queue.sql",
+        "classpath:sql/prm/profile_refresh_queue/insert_OGD_Profile.sql",
+        "classpath:sql/prm/profile_refresh_queue/insert_Solicitor_Profile.sql",
+        "classpath:sql/prm/organisation_refresh_queue/init_organisation_refresh_queue.sql",
+        "classpath:sql/prm/organisation_refresh_queue/insert_organisation1.sql",
+        "classpath:sql/prm/organisation_refresh_queue/insert_organisation2.sql"
+    })
```

Comment:
The test data SQL has a version number mismatch between profile_refresh_queue vs organisation_refresh_queue